lib/forms/find_support.yml

```diff
@@ -0,0 +1,113 @@
+_
+find_support:
+- key: are_you_out_of_employment
+  group: not_in_work
+  type: radio
+  required: true
+  options:
+    _ "yes"
+    _ "soon"
+    _ "no"
+    _ "not_sure"
+- key: are_you_non_keyworker_still_going_to_work
+  group: going_in_to_work
+  type: radio
+  required: true
```

Comment:
Could you give an example of a question that can be skipped? For example, when would it be skipped?

Reply:
We could invert the default value if that made sense, so it defaults to required if not explicitly told otherwise. But i think it might make sense to offer the change that a question has no validation on it.

I think the design system is also opinionated about skippable questions, we had a conversation on the Brexit checker about "Continue" buttons over "Next" etc. So we might need to allow the partials to add additional elements if things are skippable?

We see that [we did that here](https://www.gov.uk/transition-check/questions) - even if we might regret it somewhat, I can imagine there being calls for it.

Reply:
Ah! I assumed "skippable" means that the user is never shown the question, because they answered "no" to something else, not that they're allowed to not answer the question.

Reply:
ah right, I've been thinking of that as "criteria", so the NHS number question is dependent on you having answered Yes to do the "do you know your NHS number question". I've parked that in the to-do-later stuff, because i don't think we have any questions with dependent criteria here (we'll revisit that checkbox perhaps).

So skipptable i guess means unvalidated, you can click next / continue and not provide a response.